dtcw

```diff
@@ -14,7 +14,11 @@ set -o pipefail
 #export DTC_TEMPLATE2=https://....zip
 #...
 
-MAIN_CONFIG_FILE=docToolchainConfig.groovy
+if [[ -z "${MAIN_CONFIG_FILE:-""}" ]]; then
+  DTC_MAIN_CONFIG_FILE=docToolchainConfig.groovy
+else
+  DTC_MAIN_CONFIG_FILE="${MAIN_CONFIG_FILE}"
+fi
```

Comment:
Since the environment variables are used by the user to tweak the behavior of docToolchain we should standardize the naming conventions for the environment variables which are exposed to the user.

The other variables use the `DTC_` prefix. What about `DTC_CONFIG_FILE`?

I don't like the term "main config file" since it suggests the existence of additional configuration files. So far, I haven't seen other user exposed configuration files.

If we use `DTC_CONFIG_FILE` we should change the Gradle parameter to `-PconfigFile` too (which doesn't have to be done in this PR).

I see no point to use two variables. 

The code would look like
```sh
: ${DTC_CONFIG_FILE:-docToolchainConfig.groovy}

Reply:
looks good to me. And yes, I guess it would be good practice to prefix all our environment variables with "DTC" = _d_oc_T_ool_c_hain

Reply:
If we use `DTC_CONFIG_FILE` do change the Gradle parameter from `mainConfigFile` to `configFile`?

This way developers don't have to make a mental mapping, and it makes the code cleaner.
Since it is an internal change, we could do that in a separate issue.

Reply:
This will be a breaking change, won't it?
Yes, we should change it but I wonder if we should also support `mainConfigFile` for backwards compability without documenting it. What do you think?

Reply:
This really depends how you define the interface. Since the whole point of this issue was that the config name was not configurable it was not exposed via doctoolchain nor via dtcw.

I don't know how grade works if someone would provide the `manConfigFile` a secon time vie `DTC_OPT`.

So I see it more as this parameter was newer exposed.

But if it doesn't make much work, we could add the new parameter `configFile` which maybe overruled by the existing on and provide a warning.

But we can discuss this in issue we are going to make.

Reply:
ok. I guess you are right, we just expose it as new parameter, so let's do it!